tasks/ssh.yml

```diff
@@ -1,15 +1,18 @@
 ---
-- stat: path="{{ kdump_sshkey }}"
+- name: Check whether key exists
+  stat: path="{{ kdump_sshkey }}"
   register: sshkey_stats
```

Comment:
this does not have a problem in check mode?

Reply:
No, this works fine in check mode.

Reply:
Does it really, given what you say below?